apps/meteor/server/services/omnichannel-analytics/service.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable new-cap */
```

Comment:
Do we still need this?

Reply:
Yeah, this is because the functions inside the other classes are named with capital letters, but the rule is only `new` constructors can be with capitals so, ignoring for now 😢